extra/resources/ping

```diff
@@ -143,6 +143,7 @@ Enables to use default attrd_updater verbose logging on every call.
 <action name="monitor" depth="0"  timeout="60s" interval="10s"/>
 <action name="meta-data"  timeout="5s" />
 <action name="validate-all"  timeout="30s" />
+<action name="reload"  timeout="20s" />
```

Comment:
This will actually be reload-agent (which is for changes in agent parameters)

reload doesn't apply to ping since there's no daemon involved. reload would be for changes in a service's own configuration, for example reload for the apache web server would make it re-read httpd.conf.

You can squash this commit with the one marking parameters as reloadable, together they make the capability work